java/spacewalk-java.changes

```diff
@@ -1,5 +1,8 @@
+- fix download of packages with caret sign in the version due
+  to missing url decode
 - Add specific requirement for Cobbler 3.2.1 to not conflict with Leap 15.4
 
+
```

Comment:
Seemss you are adding an extra empty line?

Reply:
fixed